server/plugins/renderer/responses/redirect-response.js

```diff
@@ -8,7 +8,9 @@ module.exports = function (location, headers, statusCode) {
         response.statusCode = statusCode;
 
         _.each(headers, function (value, name) {
-            response.header(name, value);
+            if (name !== 'transfer-encoding') {
```

Comment:
not sure. all redirects were not responding in my machine. Only until I remove this header started working.